tools/crm_resource.c

```diff
@@ -481,12 +481,18 @@ static GOptionEntry location_entries[] = {
       "Lifespan (as ISO 8601 duration) of created constraints (with\n"
       INDENT "-B, -M) see https://en.wikipedia.org/wiki/ISO_8601#Durations)",
       "TIMESPEC" },
-    { "master", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE, &options.promoted_role_only,
+    { "promoted", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE,
+      &options.promoted_role_only,
       "Limit scope of command to promoted role (with -B, -M, -U). For\n"
       INDENT "-B and -M, previously promoted instances may remain\n"
       INDENT "active in the unpromoted role.",
       NULL },
 
+    // Deprecated since 2.1.0
+    { "master", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE,
+      &options.promoted_role_only,
+      "Deprecated: Use --promoted instead", NULL },
+
```

Comment:
In crm_mon at least, we have a deprecated_entries option group all these are added to and are displayed separately.

Reply:
Yeah at some point I want to define a policy and bring all the tools in line with it, but not going to worry about it for 2.1.0